kernel/src/schema.rs

```diff
@@ -167,6 +178,15 @@ impl StructField {
         &self.metadata
     }
 
+    /// Convert our metadata into a HashMap<String, String>. Note this copies all the data so can be
+    /// expensive for large metadata
+    pub fn metadata_with_string_values(&self) -> HashMap<String, String> {
+        self.metadata
+            .iter()
+            .map(|(key, val)| (key.clone(), val.to_string()))
+            .collect()
```

Comment:
nit: Is this short enough for clippy to leave it alone?
```suggestion
        self.metadata.iter().map(|(k, v)| (k.clone(), v.to_string())).collect()
```

Reply:
sadly not, still puts it on multiple lines